tests/test_causaleffect.py

```diff
@@ -38,7 +38,11 @@ def setUpClass(cls) -> None:
         except Exception as e:
             raise unittest.SkipTest(f"R packages not properly installed.\n\n{e}") from None
 
-        cls.assertIsNotNone(robjects, msg="make sure this was imported correctly.")
+        cls.assertIsNotNone(
+            unittest.TestCase(), robjects, msg="make sure this was imported correctly."
+        )
+        # callahanr: @cthoyt Please take a look at this, adding the first parameter is a fix to correct a mypy error:
```

Comment:
@callahanr I don't think working on typing is the best use of our time right now